app/assets/javascripts/tylium/pages/activities.js

```diff
@@ -29,4 +29,10 @@ document.addEventListener('turbolinks:load', function() {
       }
     });
   }
+
+  $(function() {
+    $(".activity-filters").change(function() {
+      $(this).children('form').submit();
+    });
+  });
```

Comment:
```suggestion
  $(function() {
    $('[data-behavior~=activity-filters]').change(function() {
      $(this).submit();
    });
  });
```
We don't use classes, IDs, element types, etc for selectors in scripts. We always use data-attributes so it's clear in the view file that this is for JS. A div with a class `activity-filters` will likely make others assume this is for CSS styling rather than a listener in a js file.

Reply:
Also changed this:
`$(this).children('form').submit();`
to this so it works:
`$(this).submit();`

Reply:
> Also changed this:
$(this).children('form').submit();
to this so it works:
$(this).submit();

Perfect, that was part of my suggested change above. 👍